code/modules/reagents/reagent_dispenser.dm

```diff
@@ -437,6 +448,7 @@
 		AM.forceMove(loc)
 
 /obj/structure/reagent_dispensers/cauldron/barrel/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	..()
```

Comment:
put this at the end so that it comes after special behaviors like grabbing

Reply:
I had a closer look at this, you've just added a second attackby for the same object! Don't do that.

Also, you're right, it is handled later, so why are you calling it here at the top??

Reply:
Because I had Shifty & Barry telling me to add a ..() at the top 🤷‍♂️

Reply:
They probably didn't see the other one at the bottom. You doubleplussure need to not have two attackbys on the same object. It works but it's actually the worst thing you can do for code legibility.